src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Windows.Tests.cs

```diff
@@ -136,6 +136,7 @@ public void TestInvalidDisksSpaceProperties()
         }
 
         [Fact, OuterLoop]
+        [ActiveIssue(1355, PlatformID.Windows)]
```

Comment:
Does this ActiveIssue need the platform ID? I'd have thought no, since there's already a platform-specific attribution on the test. I've been using the platform qualification on ActiveIssue when the issue is specific to that platform, not when the test is.